examples/check/main.go

```diff
@@ -0,0 +1,103 @@
+package main
```

Comment:
In this file I have put all the logic of the e2e client and the checks.

I have used golang native http package on purpose as this is supposed to represent an 'external' client. The 'script' is actually a rework of a bash-script that was getting a bit messy :grin: . And I realized that using go also for a small 'script' was actually so much simpler, maintainable, reliable, communicative and funnier.